layers/transportation/transportation.sql

```diff
@@ -359,9 +359,17 @@ FROM (
          FROM osm_highway_linestring
          WHERE NOT is_area
            AND
-               CASE WHEN zoom_level = 12 THEN transportation_filter_z12(highway, construction)
+               CASE WHEN zoom_level = 12 THEN
+                         CASE WHEN transportation_filter_z12(highway, construction) THEN TRUE
+                              WHEN network IN ('hike-international', 'hike-national', 'hike-regional') THEN TRUE
```

Comment:
Is this just an old code, which should be replaced? 
```suggestion
                              WHEN network IN ('iwn', 'nwn', 'rwn') THEN TRUE
```

Reply:
Yes, my version is clearly wrong here.  I need to investigate this because that network value should be coming from the route relation so I'm not sure that network is populated in that way.  Going to send this back to draft took more closely at this.

Reply:
Thank you.

As for `osm_transportation_name_linestring` table where is `route_rank` added in https://github.com/openmaptiles/openmaptiles/blob/949896a205636b25b9ca0305bcabe7ae028e2461/layers/transportation_name/update_transportation_name.sql#L57

then you are using `route_rank` directly in `layer_transportation_name()` function
https://github.com/openmaptiles/openmaptiles/blob/949896a205636b25b9ca0305bcabe7ae028e2461/layers/transportation_name/transportation_name.sql#L117


But there is no `route_rank` in `osm_highway_linestring`. And as you use this approach for populating `osm_route_member`, https://github.com/openmaptiles/openmaptiles/blob/949896a205636b25b9ca0305bcabe7ae028e2461/layers/transportation/update_route_member.sql#L102

I thought, that it could be used here as I suggested.

Reply:
The solution needed to be a bit more complex.  The most efficient approach was to use the `osm_transportation_name_network` table directly.  To achieve this, I moved that materialized view down to the `transportation` layer, added unique index for performance, and simply joined it to the z12-14 table merge in order to access the pre-calculated route rank.